tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -360,9 +360,10 @@ def test_manifest(tests, id)
           #  raise 'Errored test'
           # end
           on(tests[:proxy_agent], puppet_device_cmd, acceptable_exit_codes: [0, 1, 2])
-          if !(stdout.include? 'Applied catalog') && tests[id][:stderr_pattern].nil?
+          if tests[id][:stderr_pattern].nil? && (output[/Error: /] || !output[/Applied catalog/])
```

Comment:
This does not look right to me.  When using the `on` agentfull method you should use `stdout` instead of `output` right?

Reply:
Good catch. It's a cut&paste from the agentless codepath. I'm still setting up the agentfull TB so I'll fix & confirm there.